site/docs/reference/Connectors/capture-connectors/PostgreSQL.md

```diff
@@ -0,0 +1,113 @@
+    GRANT SELECT ON ALL TABLES IN SCHEMA information_schema, pg_catalog TO flow_capture;
+    ```
+
+    where `<others>` lists all schemas that will be captured.
+    :::info
+    If an even more restricted set of permissions is desired, you can also grant SELECT on
+    just the specific table(s) which should be captured from. The ‘information_schema’ and
+    ‘pg_catalog’ access is required for stream auto-discovery, but not for capturing already
+    configured streams.
+    :::
+3. Create the watermarks table, grant privileges, and create publication:
+
+```console
+CREATE TABLE IF NOT EXISTS public.flow_watermarks (slot TEXT PRIMARY KEY, watermark TEXT);
+
+GRANT ALL PRIVILEGES ON TABLE public.flow_watermarks TO flow_capture;
+CREATE PUBLICATION flow_publication FOR ALL TABLES;
+```
+4. Set WAL level to logical:
+```console
+ALTER SYSTEM SET wal_level = logical;
+```
+5. Restart PostgreSQL to allow the WAL level change to take effect.
+
+## Configuration
+You may set up the configuration using the `flowctl discover` workflow, or with Flow's coming UI. Either of these methods will set up a basic [capture](../../../concepts/catalog-entities/captures.md) configuration, but you must supply additional values for the PostgreSQL connector.
+
+### Values
+| Value | Name | Type | Required/Default | Details |
+|-------|------|------|---------| --------|
+|  |Tenant |String| Required | The tenant in which to create the capture. This is the prefix of your cloud storage bucket. |
+| |Name | String | Required |The unique name of the capture |
+| `connectionURI` | Database Connection URL | String | Required | Connection parameters, as a libpq-compatible connection string |
+| `max_lifespan_seconds` | Maximum Connector Lifespan (seconds) | number | 0.0 | When nonzero, imposes a maximum runtime after which to unconditionally shut down |
+| `poll_timeout_seconds` | Poll Timeout (seconds) | number | 0.0 | When tail=false, controls how long to sit idle before shutting down |
+| `publication_name` | Publication Name | string | `"flow_publication"` | The name of the PostgreSQL publication to replicate from |
+| `slot_name` | Replication Slot Name | string | `"flow_slot"` | The name of the PostgreSQL replication slot to replicate from |
+| `watermarks_table` | Watermarks Table | string | `"public.flow_watermarks"` | The name of the table used for watermark writes during backfills |
+
+### Sample
+A minimal capture definition within the catalog spec will look like the following:
+
+```yaml
+captures:
+  $tenant/CAPTURE_NAME:
+    endpoint:
+      airbyteSource:
+        image: "ghcr.io/estuary/source-postgres:dev"
+        config:
+          connectionURI: "postgres://flow:flow@localhost:5432/flow"
+          # slot_name: “flow_slot”                     # Default
+          # publication_name: “flow_publication”       # Default
+          # watermarks_table: “public.flow_watermarks” # Default
+    bindings:
+      - resource:
+        stream: ${TABLE_NAME}
```

Comment:
There should be another property here alongside `stream`, something like `namespace: ${TABLE_NAMESPACE}`

Reply:
Whoops. That's on me for being scatterbrained, thanks!